specification/cognitiveservices/data-plane/UnifedVisionRetrieval/preview/2024-03-01-preview/UnifiedVisionRetrieval.json

```diff
@@ -0,0 +1,1289 @@
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Gets or sets the key-value pairs of metadata.",
+          "type": "object",
+          "additionalProperties": {}
+        },
+        "error": {
+          "$ref": "#/definitions/ErrorResponseDetails"
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "Gets or sets the created date and time of the document.",
+          "type": "string"
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "description": "Gets or sets the last modified date and time of the document.",
+          "type": "string"
+        },
+        "userData": {
+          "description": "Gets or sets the user data for the document.",
+          "type": "object",
+          "additionalProperties": {}
+        }
+      }
+    },
+    "IngestionDocumentResponseModelCollectionApiModel": {
+      "description": "Contains an array of results that may be paginated.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IngestionDocumentResponse"
+          }
+        },
+        "nextLink": {
+          "description": "A link to the next set of paginated results, if there are more results available; not present otherwise.",
+          "type": "string"
+        }
+      }
+    },
+    "IngestionErrorDetails": {
+      "description": "Represents the ingestion error information for each document.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "innerError": {
+          "$ref": "#/definitions/IngestionInnerErrorDetails"
+        }
+      }
+    },
+    "IngestionInnerErrorDetails": {
+      "description": "Represents the ingestion inner-error information for each document.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "innerError": {
+          "$ref": "#/definitions/IngestionInnerErrorDetails"
+        }
+      }
+    },
+    "IngestionResponse": {
+      "description": "Represents the ingestion response model for the JSON document.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets the name of the ingestion.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the status of the ingestion.",
+          "enum": [
```

Comment:
'status' property enum in LRO response should contain values: Running, Succeeded, Failed, Canceled